fud2/fud-core/src/run.rs

```diff
@@ -507,6 +581,37 @@ impl<W: Write> Emitter<W> {
         Ok(())
     }
 
+    /// Emit a Ninja build command with variable list.
```

Comment:
To be more specific: `variables` is a key/value association list. (It's not just variables; it's variables *and their values*.) Maybe I'd all it `build_cmd_with_args`, to convey that more directly?

Reply:
I kind of like "vars" because it reflects the ninja term. Though, the Ninja docs occasionally uses the term "bindings" which I think probably is better.

Reply:
Just realized these are referred to as args elsewhere. Oops, yeah, then that name is better.